pkgs/os-specific/linux/musl/default.nix

```diff
@@ -26,6 +26,12 @@ let
     sha256 = "1mzxnc2ncq8lw9x6n7p00fvfklc9p3wfv28m68j0dfz5l8q2k6pp";
   };
 
+  arch = if stdenv.hostPlatform.system == "x86_64-linux"
```

Comment:
Is there an easy way to tell if this is 64 bit or 32 bit?

Reply:
yes - `stdenv.hostPlatform.isx86_64` and `stdenv.hostPlatform.isx86_32`

Reply:
Adjusted.